src/components/MDX/Sandpack/CustomPreset.tsx

```diff
@@ -74,7 +74,7 @@ const SandboxShell = memo(function SandboxShell({
   return (
     <>
       <div
-        className="shadow-lg dark:shadow-lg-dark rounded-lg"
+        className="shadow-lg dark:shadow-lg-dark rounded-2xl"
```

Comment:
makes sense tbh 😅 I've also felt the same just want someone from the core team to confirm cc: @rickhanlonii